packages/rs-sdk/tests/fetch/data_contract.rs

```diff
@@ -30,3 +30,61 @@ async fn test_data_contract_read() {
     assert!(matches!(result, Ok(Some(_))), "result: {:?}", result);
     assert_eq!(result.unwrap().unwrap().id(), id);
 }
+
+/// Given existing and non-existing data contract IDs, when I fetch them, I get exitingthe data contract.
+#[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+async fn test_data_contracts_1_ok_1_nx() {
+    use dpp::data_contract::accessors::v0::DataContractV0Getters;
+    let cfg = Config::new();
+    let id = cfg.existing_data_contract_id;
+    let nx_id = Identifier::from_bytes(&[1; 32]).expect("parse identity id");
+    let ids = vec![id, nx_id];
+
+    let mut sdk = cfg.setup_api().await;
+
+    let result = DataContract::fetch_many(&mut sdk, ids)
+        .await
+        .expect("fetch many data contracts");
```

Comment:
This seems very confusing to me. As a developer I expect when I use fetch_many not to have to know that the second field has be some specially formatted vector.

Reply:
This is a valid point. I will impl a "wrapper" method that will make it easier.

Reply:
added fetch_by_identifiers() and fetch_by_identifier() conveniance method (used in some of the tests, to have good coverage)